rules-reviewed/eap7/eap5to6/jboss-eap5,6-7-xml.windup.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```

Comment:
the file name with comma is really ugly. Please remove/replace it with something else.